src/v1/app.ts

```diff
@@ -73,14 +73,59 @@ class AppController {
     compile(srcYamlFsItem: IFsItem, srcYaml: string, kslang: string, debug: true | false | "both"): Promise<any> {
         return this.compilerService.compile(srcYamlFsItem, srcYaml, kslang, debug).then(result => {
             ga("compile", "success");
+            // clear error
+            monaco.editor.setModelMarkers(this.ui.ksyEditor.getModel(), "err check", []);
             return result;
         }, (error: CompilationError) => {
+            // ANCHOR IN-EDITOR ERROR REPORTING
+            //console.log(error.error.s$1.split(" "));
+            this.showError(error);
             ga("compile", "error", `${error.type}: ${error.error}`);
             this.errors.handle(error.error);
             return Promise.reject(error);
         });
     }
 
+    showError(error: CompilationError) {
+        console.log(error);
+        if (!error.error.s$1) return;
+
+        let path = error.error.s$1.split(" ")[0].split("/");
```

Comment:
really cool! this `error.error.s$1` parsing magic seems pretty awkward, I need to see if I can provide some more natural bindings through ScalaJS...